### DISCUSSION TITLE
Remix docs website not working

### DISCUSSION THREAD

**Discussion by code-withAshish** [remix-run/remix#7452] [Q&A] [answered]
https://remix.run/docs/en/main/file-conventions/route-files-v2 
 
I am trying to open this website from my web browser and for some reason it shows `Unexpected server error` while my internet service is fine all other things are doing fine, somehow remix website seems broken . I am from INDIA (Asia).

**Comment by offchan42** [comment, marked as answer]
In v1, the correct link is https://remix.run/docs/en/1.19.3/file-conventions/route-files-v2
In v2, the correct link is https://remix.run/docs/en/main/file-conventions/routes
